spec/awful/completion_spec.lua

```diff
@@ -0,0 +1,52 @@
+local compl = require("awful.completion")
+local shell = function(...)
+    local command, pos, matches = compl.shell(...)
+    return {command, pos, matches}
+end
+local gfs = require("gears.filesystem")
+
+describe("awful.completion.shell in empty directory", function()
+    local orig_popen = io.popen
+
+    setup(function()
+        gfs.mkdir('/tmp/awesome-tests/empty')
+        io.popen = function(...)
```

Comment:
Hmm.. instead it would overwrite / restore the `completion` module, so there is not much gained after all.
Will add ignore statements.